app_dart/lib/src/request_handlers/update_agent_health.dart

```diff
@@ -45,13 +50,38 @@ class UpdateAgentHealth extends ApiRequestHandler<UpdateAgentHealthResponse> {
         throw BadRequestException('Invalid agent ID: $agentId');
       },
     );
+    final TabledataResourceApi tabledataResourceApi = await config.createTabledataResourceApi();
+    final List<Map<String, Object>> tableDataInsertAllRequestRows = <Map<String, Object>>[];
```

Comment:
Nit: This var name is too long.

Reply:
Changed [tableDataInsertAllRequestRows] to [requestRows].